include/miopen/config.h.in

```diff
@@ -77,6 +77,7 @@
 #cmakedefine EXTRACTKERNEL_BIN "@EXTRACTKERNEL_BIN@"
 #cmakedefine MIOPEN_OFFLOADBUNDLER_BIN "@MIOPEN_OFFLOADBUNDLER_BIN@"
 #cmakedefine MIOPEN_CACHE_DIR "@MIOPEN_CACHE_DIR@"
+#cmakedefine MIOPEN_LIBMLIRMIOPEN "@MIOPEN_LIBMLIRMIOPEN@"
```

Comment:
This path is never used as a path in the program. Please deliver `MIOPEN_USE_MLIR` from CMake and use here:
```
#cmakedefine01 MIOPEN_USE_MLIR
```
Then please replace all things like
```
#ifdef MIOPEN_LIBMLIRMIOPEN
```
with 
```
#if MIOPEN_USE_MLIR
```
Thanks.

Reply:
So effectively `#ifdef` does not need this `#cmakedefine` to work in code. Instead you are suggesting on a simpler way to actually use it by `#cmakedefine01`? If that's the case, I will update the cmakedefine usage as you suggested. Thanks for observing this.

Reply:
Yes. And we'll have macro that is _always defined_, with value 0 or 1, which will make our code more robust.